src/main/java/com/nix/events/EventsController.java

```diff
@@ -0,0 +1,57 @@
+package com.nix.events;
+
+import com.nix.events.dao.Event;
+import com.nix.events.exceptions.EventNotFoundException;
+import com.nix.events.repositories.EventRepository;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+
+@RestController
+@CrossOrigin(origins = "http://localhost:3000")
```

Comment:
This will not work inside your container/kubernetes cluster
Because when you run your service inside a container, the address changes for other services. If the container in which you run the service is called "frontend". Then the address will be the following: "http://frontend:3000"
Therefore, it is better to put the address as a variable.